src/Make/MakeMethod.php

```diff
@@ -23,13 +23,21 @@ public static function getCommandName(): string
         return 'make:method';
     }
 
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getCommandDescription(): string
+    {
+        return 'Creates a new JSON-RPC method';
+    }
+
     /**
      * {@inheritdoc}
      */
     public function configureCommand(Command $command, InputConfiguration $inputConf)
     {
         $command
-            ->setDescription('Creates a new JSON-RPC method')
```

Comment:
We need save this for previous version compatibility

Reply:
Fix